<?xml version="1.0"?>
<launch>
    <arg name="quad_name" default="blackbird"/>
    <arg name="use_bem_propeller_model" default="False"/>
    <arg name="use_unity" default="False"/>
    <arg name="debug" default="False"/>
    <arg name="render" default="False"/>
    <arg name="race_track" default="real_track"/>
    <arg name="save_data" default="false"/>
    <param name="/use_sim_time" value="True"/>
    <arg name="real_time_factor" value="1.0"/>
    <arg name="gui" default="True"/>
    <arg name="rviz" default="True"/>
    <arg name="mock_vio_params_file" default="$(find agilib)/params/mock_vio.yaml"/>

    <group ns="$(arg quad_name)">
        <node name="agiros_pilot" pkg="environment" type="environment_node" output="screen">
            <param name="param_dir" value="$(find agilib)/params"/>
            <param name="ros_param_dir" value="$(find environment)/parameters"/>
            <param name="use_bem_propeller_model" value="$(arg use_bem_propeller_model)"/>
            <param name="pilot_config" value="simple_sim_pilot.yaml"/>
            <param name="race_config" value="$(find environment)/parameters/tracks/$(arg race_track).yaml"/>
            <param name="camera_config" value="$(find environment)/parameters/camera.yaml"/>
            <param name="real_time_factor" value="$(arg real_time_factor)"/>
            <param name="render" value="$(arg render)"/>
        </node> -->
    </group>

</launch>